src/app/core/store/notifications/notifications.action.ts

```diff
@@ -42,5 +43,12 @@ export namespace NotificationsAction {
     }
   }
 
-  export type All = Error | Success;
+  export class Confirm implements Action {
+    public readonly type = NotificationsActionType.CONFIRM;
```

Comment:
That would work. 
Originally I implemented for consistency (so that the components only dispatch actions in order to do something.
Good point about usage of  `Error` and `Success` in effects. What would you say about adding the notification confirm inside the `Remove` effect? That way we would have consistent deletion of documents across the whole application?

Reply:
What a good idea! But we need to handle it the Redux way. So I would suggest creating new `DeleteConfirm` action in `DocumentsAction` namespace. Take a look at PR #202.

But please do not change the other parts of the store for now.

Reply:
A new issue with this has arisen. Getting the information, that user has clicked `yes` on confirm is impossible now.

Reply:
Why do you need that? If you remove something from the given store, the related observable will emit a new value and the UI will be automatically refreshed.

Reply:
The UI will be refreshed only on query change. When document gets deleted from store I have to delete it from layout too. And to do that I need to know what was deleted.

After deleting a document some of my page subscriptions will send new data.. but I can't use that to figure out what to delete.

Currently I found a workaround by making 1 more subscription on all documents and each time any change happens, compare ID-s of received documents to those in the layout.. and if there are any ID-s in layout that are not received document ids, remove them.
^
This sounds a bit like overkill just to figure out what got deleted. But at least it's not fetching anything from server and using only what's in store.